website/blog/2023-09-13-reason-react-ppx.md

```diff
@@ -0,0 +1,53 @@
+---
+title: reason-react-ppx is on opam!
+---
+
+ReasonReact 0.12 provides the JSX transformation from `reason-react-ppx` (previously named `reactjs-jsx-ppx`) with a bunch of improvements.
+
+The biggest changes are the new JSX transformation from React 17 and a great improvement on locations for props. A few bug fixes and some small breaking changes are also included (check below for more details).
+
+### Locations from domProps in action (Thanks @jchavarri)
+![download](https://github.com/reasonml/reason-react/assets/3763599/db505599-8fee-4889-80a3-d2056ece291c)
+
+### new JSX transform (Thanks @anmonteiro)
+<img width="1508" alt="Screenshot 2023-09-13 at 16 11 17" src="https://github.com/reasonml/reason-react/assets/3763599/b7fec116-131a-48cd-8be1-1496544131d1">
+
+## What's changing?
+
+Track `reacjs-jsx-ppx` in the same repository as `reason-react`, and rename it to `reason-react-ppx`, a more suitable name with a few improvements:
```

Comment:
would be nice to link the issues  / PRs that fixed these

Reply:
The full changelog is down, do you want the repetition?